spartan/releases/rough-rhino/full-node.sh

```diff
@@ -0,0 +1,31 @@
+docker run --rm --network=host \
+  -e P2P_UDP_ANNOUNCE_ADDR=your.ip.address:40400 \
+  -e P2P_TCP_ANNOUNCE_ADDR=your.ip.address:40400 \
+  -e COINBASE=0xbaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa \
+  -e DEBUG="aztec:*,-aztec:avm_simulator*,-aztec:circuits:artifact_hash,-aztec:libp2p_service,-json-rpc*,-aztec:world-state:database,-aztec:l2_block_stream*" \
+  -e LOG_LEVEL=debug \
+  -e AZTEC_PORT=8080 \
+  -e P2P_ENABLED=true \
+  -e VALIDATOR_DISABLED=true \
+  -e L1_CHAIN_ID=1337 \
+  -e PROVER_REAL_PROOFS=true \
+  -e PXE_PROVER_ENABLED=true \
+  -e ETHEREUM_SLOT_DURATION=12sec \
+  -e AZTEC_SLOT_DURATION=36 \
+  -e AZTEC_EPOCH_DURATION=32 \
+  -e AZTEC_EPOCH_PROOF_CLAIM_WINDOW_IN_L2_SLOTS=13 \
+  -e ETHEREUM_HOST=http://35.221.3.35:8545 \
+  -e BOOTSTRAP_NODES=enr:-Jq4QKIJisajcICBVMoMwFtbmPgmHt3KoonypbBIQCAMNjhMc6DKW0J4vJzDpGPFUX7T2fzyyjezHgKKzeZY_DbRz_kGjWF6dGVjX25ldHdvcmsBgmlkgnY0gmlwhCPdAyOJc2VjcDI1NmsxoQK92C7GObzDvCt9uwzW0lhKJKGCvOWkmAZjd2E2w-svuoN0Y3CCndCDdWRwgp3Q \
+  -e REGISTRY_CONTRACT_ADDRESS=0x5fbdb2315678afecb367f032d93f642f64180aa3 \
+  -e GOVERNANCE_PROPOSER_CONTRACT_ADDRESS=0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0 \
+  -e FEE_JUICE_CONTRACT_ADDRESS=0xe7f1725e7734ce288f8367e1bb143e90bb3f0512 \
+  -e ROLLUP_CONTRACT_ADDRESS=0x2279b7a0a67db372996a5fab50d91eaa73d2ebe6 \
+  -e REWARD_DISTRIBUTOR_CONTRACT_ADDRESS=0x5fc8d32690cc91d4c39d9d3abcbd16989f875707 \
+  -e GOVERNANCE_CONTRACT_ADDRESS=0xcf7ed3acca5a467e9e704c703e8d87f634fb0fc9 \
+  -e COIN_ISSUER_CONTRACT_ADDRESS=0xdc64a140aa3e981100a9beca4e685f962f0cf6c9 \
+  -e FEE_JUICE_PORTAL_CONTRACT_ADDRESS=0x0165878a594ca255338adfa4d48449f69242eb8f \
+  -e INBOX_CONTRACT_ADDRESS=0xed179b78d5781f93eb169730d8ad1be7313123f4 \
+  -e OUTBOX_CONTRACT_ADDRESS=0x1016b5aaa3270a65c315c664ecb238b6db270b64 \
+  -e P2P_UDP_LISTEN_ADDR=0.0.0.0:40400 \
+  -e P2P_TCP_LISTEN_ADDR=0.0.0.0:40400 \
+  aztecprotocol/aztec:698cd3d62680629a3f1bfc0f82604534cedbccf3-x86_64 start --node --archiver --sequencer --pxe
```

Comment:
want me to tag this as rough rhino?

Reply:
Oh that would be nice! Can you push a commit to this PR when you've done it?

Reply:
done